staging/src/k8s.io/kubectl/pkg/cmd/autoscale/autoscale.go

```diff
@@ -134,7 +134,7 @@ func NewCmdAutoscale(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *
 // Complete verifies command line arguments and loads data from the command environment
 func (o *AutoscaleOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []string) error {
 	var err error
-	o.dryRun = cmdutil.GetFlagBool(cmd, "dry-run")
+	o.dryRun = cmdutil.GetDryRunFlag(cmd)
```

Comment:
I'm still really not excited about this name, since we want "dry-run" to mean both client and server. And this name is based on the previous understanding that we're trying to deprecate.

Reply:
Renamed